src/schema/manifest-schema.json

```diff
@@ -17,12 +17,14 @@
                         "strict_min_version": {
                             "default": "42a1",
                             "description": "Minimum version of Gecko to support. Defaults to '42a1'. (Requires Gecko 45)",
-                            "type": "string"
+                            "type": "string",
+                            "pattern": "^[0-9]{1,3}(.[a-z0-9]+)+$"
                         },
                         "strict_max_version": {
                             "default": "*",
                             "description": "Maximum version of Gecko to support. Defaults to '*'. (Requires Gecko 45)",
-                            "type": "string"
+                            "type": "string",
+                            "pattern": "^[0-9]{1,3}(.[a-z0-9*]+)+$"
```

Comment:
Is the `.` correct or should that be escaped? This would allow `9#foo` for example.